### PR TITLE
Only allow actual member functions on member invokers

### PR DESCRIPTION
**Pull request by fpbouchard**

### CODE REVIEW

File: src/invokers.ts [hunk 1]
```diff
@@ -82,7 +81,7 @@ export function makeResolverInvoker<T>(resolver: Resolver<T>) {
    * @param  {MethodName} methodToInvoke
```
Comment (jeffijoe) [inline src/invokers.ts:81]: Just to not be inconsistent:

```suggestion
   * @param  methodToInvoke
```

**Comment by fpbouchard** [comment]
Is this PR likely to be merged? I'm open for discussions if there is something I missed.

Thanks!

**Comment by jeffijoe** [comment]
Trying to release but run into some logistical issues with Travis. This might take a while. OSS isn't as fun as it once was. 😞

**Comment by jeffijoe** [comment]
Released as v4.0 since package updates required bumping minimum Node version. Thanks for the PR

**Pull request merged**